dpnp/dpnp_array.py

```diff
@@ -97,6 +97,13 @@ def T(self):
         else:
             return dpnp.transpose(self)
 
+    def to_device(self, target_device):
+        """
+        Transfer array to target device
+        """
+
+        return dpnp_array(shape=self.shape, buffer=self.get_array().to_device(target_device))
```

Comment:
Other option is usage of method `__new__` to create `dpnp_array`. As an example there is method `flatten`.